src/assets/main.css

```diff
@@ -2,6 +2,31 @@
 
 html, body {
     font-family: 'Roboto', sans-serif;
+    margin: 0 auto;
+
+}
+body {
+    margin: 0 0 0 62px;
+    background-color: #f4f6fc;
+    color: rgb(40, 54, 75);
+}
+
+div, span, applet, object, iframe,
```

Comment:
I would leave a comment here on why we are setting these values for all the CSS elements